esp_delta_ota/examples/https_delta_ota/CMakeLists.txt

```diff
@@ -0,0 +1,7 @@
+# The following lines of boilerplate have to be in your project's CMakeLists
+# in this exact order for cmake to work correctly
+cmake_minimum_required(VERSION 3.5)
+list(APPEND EXTRA_COMPONENT_DIRS $ENV{IDF_PATH}/examples/common_components/protocol_examples_common)
```

Comment:
This will unfortunately not work when the example is installed via the component manager.

Instead, please add the esp_delta_ota as a dependency in idf_component.yml of the example, and specify override_path so that the local version of the component is used when the example is built from a git repository.

Reply:
Hello Ivan,
This line appends `protocol_examples_common` component which is present in IDF and required to connect to WiFi.

Also, I have already added esp_delta_ota as a dependency (with override path) in idf_component.yml file of the example. So it won't be a problem to build it when it is downloaded using the component manager.